File: add/codesnippet/CPP/p-system.configuration.i_12_1.cpp
   AssemblyInstaller^ myAssemblyInstaller = gcnew AssemblyInstaller;
   ServiceInstaller^ myServiceInstaller = gcnew ServiceInstaller;
   EventLogInstaller^ myEventLogInstaller = gcnew EventLogInstaller;

   InstallerCollection^ myInstallerCollection = myAssemblyInstaller->Installers;
   
   // Add Installers to the InstallerCollection of 'myAssemblyInstaller'.
   myInstallerCollection->Add( myServiceInstaller );
   myInstallerCollection->Add( myEventLogInstaller );

   array<Installer^>^ myInstaller = gcnew array<Installer^>(2);
   myInstallerCollection->CopyTo( myInstaller, 0 );
   // Show the contents of the InstallerCollection of 'myAssemblyInstaller'.
   Console::WriteLine( "Installers in the InstallerCollection : " );
   for ( int iIndex = 0; iIndex < myInstaller->Length; iIndex++ )
      Console::WriteLine( myInstaller[ iIndex ]->ToString() );

File: add/codesnippet/CSharp/p-system.configuration.i_12_1.cs
      AssemblyInstaller myAssemblyInstaller = new AssemblyInstaller();
      ServiceInstaller myServiceInstaller = new ServiceInstaller();
      EventLogInstaller myEventLogInstaller = new EventLogInstaller();
      
      InstallerCollection myInstallerCollection = myAssemblyInstaller.Installers;
      
      // Add Installers to the InstallerCollection of 'myAssemblyInstaller'.
      myInstallerCollection.Add(myServiceInstaller);
      myInstallerCollection.Add(myEventLogInstaller);

      Installer[] myInstaller = new Installer[2];
      myInstallerCollection.CopyTo(myInstaller,0);
      // Show the contents of the InstallerCollection of 'myAssemblyInstaller'.
      Console.WriteLine("Installers in the InstallerCollection : ");
      for (int iIndex=0; iIndex < myInstaller.Length; iIndex++)
         Console.WriteLine(myInstaller[iIndex].ToString());